CONTRIBUTING.md

```diff
@@ -383,6 +383,37 @@ tested it becomes an officially supported Akka feature.
 
 [List of Akka features marked as may change](http://doc.akka.io/docs/akka/current/common/may-change.html)
 
+## Java APIs
+
+Each feature in Akka must have an API for convenient usage from both Scala and Java, as we develop the Java APIs in
+a subset of Scala that is easy to use from Java there are ... 
```

Comment:
'as we develop ...' can perhaps be dropped entirely, as we go into more detail below anyway?